### DISCUSSION TITLE
How to see git output history for session?

### DISCUSSION THREAD

**Discussion by Wiebesiek** [tpope/vim-fugitive#2182] [General]
Is there a way to see more than output from the last git command? I was able to find that `:Gedit -` will bring up the output from the last command, but is there a way to see the output from the entire session?

**Comment by tpope** [comment]
Every command is captured to its own temporary file. `:Gedit -` opens up the most recent temporary file. You can open up older temporary files by name, but Fugitive doesn't provide anything special to facilitate this.

There's no such thing as a "session" in Fugitive; each command is its own standalone event. So "output from the entire session" isn't a supported operation.
